ehrapy/api/_anndata_util.py

```diff
@@ -36,3 +37,41 @@ def get_column_values(adata: AnnData, indices: Union[int, List[int]]) -> np.ndar
         :class:`~numpy.ndarray` object containing the column values
     """
     return np.take(adata.X, indices, axis=1)
+
+
+def get_numeric_vars(adata: AnnData) -> List[str]:
+    """Fetches the column names for numeric variables in X
```

Comment:
```suggestion
    """Fetches the column names for numeric variables in X.
```

It is convention to always end the header sentence with a `.` :) I disabled the linting checks for this because I don't want it to be pedantic, but just mentioning it once.